decoder/Cargo.toml

```diff
@@ -42,6 +42,7 @@ object = { version = "0.32", default-features = false, features = [
 ] }
 serde = { version = "1", features = ["derive"] }
 serde_json = { version = "1", features = ["arbitrary_precision"] }
+alterable_logger = "1.0.0"
```

Comment:
What was this change for?

Reply:
It's for lines 140 and 141 in `decoder/src/log/mod.rs`, since what is in this PR the `run` method (it was `main` before) is calling `init_logger` on each restart - meaning ELF file change. But since a logger is already set after the first time calling `init_logger` will crash after the first reload